src/main/java/com/metamx/emitter/service/AlertBuilder.java

```diff
@@ -66,9 +67,9 @@ public AlertBuilder severity(AlertEvent.Severity severity)
     return new ServiceEventBuilder<AlertEvent>()
     {
       @Override
-      public AlertEvent build(String service, String host)
+      public AlertEvent build(ImmutableMap<String, String> serviceDimensions)
```

Comment:
Its used in `io.druid.server.coordinator.DruidCoordinatorRuleRunnerTest#testRunTwoTiersTierDoesNotExist` and other places for mocking purposes. I suggest keeping it as interface unless you can guarantee those mocks won't fail.